.github/SECURITY.md

```diff
@@ -0,0 +1,3 @@
+If you believe that there is a security vulnerability in PowerShell,
+it **must** be reported to [secure@microsoft.com](https://technet.microsoft.com/security/ff852094.aspx) to allow for [Coordinated Vulnerability Disclosure](https://technet.microsoft.com/security/dn467923).
+**Only** file an issue, if [secure@microsoft.com](https://www.microsoft.com/en-us/msrc/faqs-report-an-issue?rtc=1) has confirmed filing an issue is appropriate.
```

Comment:
We have had cases where an issue was rejected because it was breaking and then it was asked to be reconsidered on security ground.  

Should we add a statement that asks to consider an existing issue as security vulnerability should go through secure@ as well?

Reply:
This decision is about to you if you want to further change/enhance the text